services/tenant-ui/frontend/src/components/innkeeper/config/ServerConfig.vue

```diff
@@ -0,0 +1,242 @@
+        {{ serverConfig?.config?.version }}
+      </p>
+      <p>
+        <strong>{{ $t('serverConfig.tractionProxy') }}</strong>
+        {{ config.frontend.tenantProxyPath }}
+        <a :href="swaggerUrl" target="_blank">
+          <i class="pi pi-external-link ml-2" />
+        </a>
+      </p>
+
+      <div class="grid mt-3">
+        <div class="col">
+          <!-- Traction cfg - ledger -->
+          <Fieldset
+            :legend="$t('serverConfig.ledger.ledgerDetails')"
+            :toggleable="true"
+          >
+            <p class="mt-0">
+              <strong>{{ $t('serverConfig.ledger.ledgerList') }}</strong>
+            </p>
+            <DataTable :value="ledgerConfigList" size="small" striped-rows>
+              <Column field="id" header="id"></Column>
+              <Column field="endorser_alias" header="endorser_alias"></Column>
+              <Column field="endorser_did" header="endorser_did"></Column>
+            </DataTable>
+            <p>
+              <strong>{{ $t('serverConfig.ledger.quickConnect') }}</strong>
+              {{ config.frontend.quickConnectEndorserName }}
+            </p>
+            <p>
+              <strong>{{ $t('serverConfig.ledger.default') }}</strong>
+              {{ serverConfig?.config?.['ledger.write_ledger'] }}
+            </p>
+          </Fieldset>
+
+          <!-- Traction cfg - storage -->
+          <Fieldset
+            class="mt-4"
+            :legend="$t('serverConfig.storage.title')"
+            :toggleable="true"
+          >
+            <p class="mt-0">
+              <strong>{{
+                $t('serverConfig.storage.walletStorageConfig')
+              }}</strong>
+              {{ serverConfig?.config?.['wallet.storage_config'] }}
+            </p>
+            <p>
+              <strong>{{
+                $t('serverConfig.storage.walletStorageType')
+              }}</strong>
+              {{ serverConfig?.config?.['wallet.storage_type'] }}
+            </p>
+            <p>
+              <strong>{{ $t('serverConfig.storage.walletType') }}</strong>
+              {{ serverConfig?.config?.['wallet.type'] }}
+            </p>
+          </Fieldset>
+
+          <!-- Plugins from API call to get them -->
+          <PluginList class="mt-4" />
+
+          <!-- Traction cfg - raw json -->
+          <Accordion class="mt-4">
+            <AccordionTab :header="$t('serverConfig.expand')">
+              <div v-if="loading" class="flex justify-content-center">
+                <ProgressSpinner />
+              </div>
+              <vue-json-pretty v-else :data="serverConfig" />
+            </AccordionTab>
+          </Accordion>
+        </div>
+
+        <div class="col">
+          <!-- Details about tenants/reservations -->
+          <Fieldset
+            :legend="$t('serverConfig.tenants.title')"
+            :toggleable="true"
+          >
+            <p class="mt-0">
+              <strong>{{ $t('serverConfig.tenants.token') }}</strong>
```

Comment:
Could probably have a reusable component for these fields. Would make this component cleaner.

Reply:
You thinking a reusable component for the `p>strong>field name: fieldvalue` pairs with this?
Or breaking the sub panel contents into their own components to keep the ServerConfig page smaller?

(both sound like reasonable ideas)

Reply:
I was just thinking the `<p>strong>field name: fieldvalue` when I wrote this. Just because I think it would look cleaner having so many strong and curly brace elements. You could make the value prop undefinable as well and avoid all the `?'s` in this component.

Reply:
Extracted to a `ConfigItem` component that can take in text (or other simple types) as props, or slots if HTML (icons, links etc) want to be provided. Added the `:` delimiter to i18n so it can be changed in one spot if we ever needed to change up the format. Can add this to a bunch of other places around the app as we go.
https://github.com/bcgov/traction/pull/914/commits/dcb4c2c477c3b3bf5444cf459bff417633a83223

Reply:
Nice! That looks better.